docs: data plane signaling blog article (#144)
eclipse-edc/eclipse-edc.github.io: content/en/blog/20260126-data-plane-signaling/index.md

the EDC Data Plane Framework.

## Rationale

The Data Plane Signaling specification addresses key architectural ambiguities by clearly delineating the roles of the
Control Plane (orchestration and protocol-level authentication) and the Data Plane (actual data transfer and flow-specific
authentication). This separation resolves design confusion, establishes symmetry between push and pull patterns, and,
most importantly, creates a standard interface. This allows any application to become a compliant Data Plane,
fostering innovation and moving the industry toward a future where data sharing is a seamless, integrated layer within
sovereign applications, rather than a connector-centric task.

## Approach

The implementation of the Data Plane Signaling protocol in the EDC Control Plane is already progressing
([issue](https://github.com/eclipse-edc/Connector/issues/5323)), we expect to have a fully compliant version as soon as
the Data Plane Signaling specification will be released in a stable manner.

At that point, we will deprecated the EDC Data Plane Framework in its entirety, and after 2 releases, it will be eligible

> for removal from the codebase, so if you are an EDC adopter we recommend you to start prototyping your Data Plane(s) 

Reviewer: ```suggestion
for removal from the codebase. If you are an EDC adopter, we recommend to start prototyping your Data Plane(s) 
```

following the current specification. The dataplane-signaling team is already providing SDKs library in different

> languages (Go, Rust, Java, .Net) so you can focus only on the actual data flow management.

Reviewer: ```suggestion
languages (Go, Rust, Java, . NET) to help with this transition.
```
